spec/reek/smells/attribute_spec.rb

```diff
@@ -40,210 +43,80 @@
           attr :super_thing2
           private
           attr :super_thing2
-        end"
-      end
-
-      it_should_behave_like 'no attribute found'
+        end
+      ').to_not reek_of(:Attribute)
     end
 
-    context 'declared in a module' do
-      before :each do
-        @src = "module Fred
```

Comment:
i'm not sure how we should handle this. the only code that is related to class/module is `self.contexts`. the actual implementation does not rely on classes or modules at all. we need a test to make sure that the smell is running in the context of a module or a class, is there a better way in reek to do this? 

before it felt like there is something special happening for classes and modules. that gave me the impression to duplicate all tests cases for modules. which in fact doesn't really make sense.